### ISSUE TITLE
Missing Dependency 'renvm-gateway'

### ISSUE THREAD

**Issue by wmcclinton** [renproject/multichain#135]
Hello, I got the error: failed to load source for dependency 'renvm-gateway'

I believe that this repo is a dependency and is missing: https://github.com/renproject/ren-solana

**Comment by jazg** [comment]
Hey @wmcclinton, can you please explain how to reproduce the issue?

**Comment by wmcclinton** [comment]
Thanks @jazg, I cloned this repository and tried to build the Dockerfile to an image using my github credentials. It then gave the error: failed to load source for dependency 'renvm-gateway'

Also github.com/renproject/ren-solana doesn't exist  and  it is being used in the Dockerfile on line 27: ENV GOPRIVATE="github.com/renproject/ren-solana, github.com/renproject/solana-ffi"

**Comment by jazg** [comment]
@wmcclinton What is your use case? This Dockerfile is designed specifically for Darknode/Lightnode. If you're looking to run chain specific nodes, you can check out `/infra/<chain>/Dockerfile`

**Comment by jazg** [comment]
@wmcclinton Yep, Multichain itself isn't dependent on these components of RenVM, so you shouldn't need to run any of these packages to test your code. The chain-specific packages just need to implement certain interfaces for addresses, gas, and transactions. They can be tested by running your chain node locally (using the Dockerfile you create) and running the corresponding unit tests.

**Comment by jazg** [comment]
Keep in mind that adding support for your chain in Multichain does not guarantee it will be supported by the Darknodes, it just means it'll be easier for them should they choose to do so (e.g. if there is evidence that there will be volume flowing through this chain).
